src/modules/lua_general.c

```diff
@@ -941,7 +943,8 @@ mtev_console_state_lua(mtev_console_closure_t ncct,
     return -1;
   }
   if(argc) {
-    eventer_set_owner(ncct->e, eventer_choose_owner(atoi(argv[0])));
+    if(ncct->simple)
+      eventer_set_owner(ncct->simple->e, eventer_choose_owner(atoi(argv[0])));
```

Comment:
In honor of the recent linux `sudo` bug, do we trust `argv[0]` enough here that we don't need to check the return value?